Updating guide, adding location instructions

ig-perez opened the pull request:
Added instructions about how to change device location data for Alexa and Google.

ig-perez committed:
Updating guide, adding location instructions

dmarvp commented on end-to-end/guide.md:
```diff
@@ -113,8 +113,7 @@ Below the end-to-end testing configuration options and what they do are listed:
 | [locales](#locales) | The locale or locales to be used - a comma-delimited list. The entire suite will be run once for each locale |
-| locationLat | The latitude in degrees from where the requests should be made. It must be in the range [-90.0, +90.0]. Only for Google virtual devices |
-| locationLong | The longitude in degrees from where the requests should be made. It must be in the range [-180.0, +180.0] Only for Google virtual devices |
+| deviceLocation | An object to set a specific location value from where the requests are sent in Google Virtual Devices. It is composed by the `lat` and `lng` keys. Read [here](./faq#how-do-i-change-my-virtual-device-location-to-test-location-specific-features) to know how to use it.
```
to follow the style of the other properties, put the link on `deviceLocation` instead of "Read here".

ig-perez replied:
ok

jkelvie commented:
One question - what is the interplay between the main account address on Amazon and the device account? Is the account address used if one is not set on the device? Which takes precedence?

I would like to know, and our users probably would as well.

dmarvp commented on end-to-end/faq.md:
```diff
@@ -259,6 +259,29 @@ By default, our virtual devices will always point to the USA region. If you want
 
 2. For Google: Use the properties locationLat and locationLong on your test scripts or `testing.json` file to specify the latitude and longitude from where your virtual device should be located in order to test your voice app.
 
+## **How do I change my Virtual Device location to test location-specific features?**
```
How do I change my default Virtual Device location to test location-specific features?

ig-perez replied:
Guess better not to use default here.

dmarvp commented on end-to-end/faq.md:
```diff
@@ -259,6 +259,29 @@ By default, our virtual devices will always point to the USA region. If you want
 2. For Google: Use the properties locationLat and locationLong on your test scripts or `testing.json` file to specify the latitude and longitude from where your virtual device should be located in order to test your voice app.
 
+## **How do I change my Virtual Device location to test location-specific features?**
+Our Virtual Devices are created without a specific location. If you want to set it, follow the instructions below:
```
> Our Virtual Devices are created without a specific location.

That's not entirely true. Even if we don't set one by default, both Alexa and Google will infer a location from your IP, account region, etc. To make it simpler, I would just remove this line and go directly to the instructions.

ig-perez replied:
Yeah, better

dmarvp commented:
@ig-perez , we could answer @jkelvie's question in another faq right before this one. `What is the default location for a virtual device?` and then proceed to explain for Alexa and Google. You can find the precedence for Google here:

![image](https://user-images.githubusercontent.com/6411740/74958914-b0f2d600-53d7-11ea-8d73-fb1323f38a1d.png)

For Amazon, we need to test with accounts from other regions and then changing a device location.

ig-perez commented:
> 
> 
> @ig-perez , we could answer @jkelvie's question in another faq right before this one. `What is the default location for a virtual device?` and then proceed to explain for Alexa and Google. You can find the precedence for Google here:
> 
> ![image](https://user-images.githubusercontent.com/6411740/74958914-b0f2d600-53d7-11ea-8d73-fb1323f38a1d.png)
> 
> For Amazon, we need to test with accounts from other regions and then changing a device location.

Good idea, I'll add the information on another question, but I partially explained for Amazon in current FAQ question.

ig-perez commented:
> 
> 
> One question - what is the interplay between the main account address on Amazon and the device account? Is the account address used if one is not set on the device? Which takes precedence?
> 
> I would like to know, and our users probably would as well.

@jkelvie yes, it takes the address from your main Amazon account as I wrote in the instructions, but I'll give more details on a separate question based on Diego's suggestion

ig-perez committed:
Adding more information about default location for virtual devices

ig-perez commented:
@dmarvp @jkelvie , I have added an extra question about the default location for virtual devices. Oddly, I can't reproduce the behavior I saw yesterday with my account (I deleted my current address and a virtual device took previous address for location). For US accounts, Atlanta and San José are used when creating virtual devices, no matter if the Amazon account has an address or not. This behavior was not seen for the UK.

jkelvie commented:
One clarification - when outside the US, if no address is set on the device, does it the use the account address as a fallback?

ig-perez commented:
@jkelvie no, in the UK the device address was always empty, and when asking for the weather alexa responded "add an address"-like message

jkelvie commented:
Okay, please make note of that behavior in the FAQ. I think it is helpful to have it memorialized since it comes up as a question somewhat frequently.

ig-perez committed:
Adding more details to location questions

ig-perez commented:
@jkelvie I added more information as requested, please let us know if it is clear now.

ig-perez committed:
Changing "Bespoken Tools" to "Bespoken"
